src/frontend/web/src/ratingpage/EffortBar.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import './EffortBar.css';
+
+const EffortBar = (props) => {
```

Comment:
It is nicer to receive the props already split to their content - in this case `{effort}` so you could use them without the prefix `props.` in the rest of the file.
```suggestion
const EffortBar = ({effort}) => {
```

Reply:
Good to know. Thanks!